rn/Teacher/src/modules/inbox/CourseSelect.js

```diff
@@ -110,6 +110,7 @@ export function mapStateToProps (state: AppState): CourseSelectDataProps {
   let courses = Object.keys(state.entities.courses)
     .map(id => state.entities.courses[id].course)
     .filter(App.current().filterCourse)
+    .filter(course => !course.access_restricted_by_date)
```

Comment:
I don't know if I should just add this to `App.current().filterCourse`. The Dashboard makes heavy use of it and I don't know if there's ever a case where we'd show a course that has `access_restricted_by_date = true` on the Dashboard, all courses, invites, etc. Any thoughts?

Reply:
Yeah, honestly I'd rather they not come back in the api at all. As far as I know, we never show a restricted course. The earlier we can filter them out the better.